### PR TITLE
Fix #765: Added generated VCARD4 vocabulary

### LINKED ISSUE #765: Support for VCARD4 vocabulary
I'm working on a https://www.w3.org/TR/vcard-rdf/ aka VCARD4 vocabulary class (used in a few EU-supported vocabularies like DCAT-AP, Joinup.eu Core Vocabularies)

### PR DESCRIPTION
**Pull request by barthanssens**
Signed-off-by: Bart Hanssens <bart.hanssens@fedict.be>

This PR addresses GitHub issue: #765  .

Briefly describe the changes proposed in this PR:

* 
* 
*

### CODE REVIEW

**Comment by jeenbroekstra** [comment]
Everyone: given that my involvement with the project is currently a lot less than it was, I have less time to do proper PR reviews, so  I'm relying on your explicit input. Please leave comments or reviews on any pull requests, indicating if you are happy for it to be merged or not. 

I will not merge any pull request unless I see a comment/review from at least one other reviewer that he/she is ok with a merge.

File: core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/VCARD4.java [hunk 1]
```diff
@@ -0,0 +1,658 @@
...
+import org.eclipse.rdf4j.model.ValueFactory;
+import org.eclipse.rdf4j.model.impl.SimpleNamespace;
+import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
+
+
+/**
+ * Constants for the vCard Ontology.
+ *
+ * @see <a href="https://www.w3.org/TR/2014/NOTE-vcard-rdf-20140522/">vCard Ontology</a>
+ *
```
Comment (jgrzebyta) [inline core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/VCARD4.java]: I would add URI of realisation file: either *.owl or *.rdf.

**Comment by jgrzebyta** [comment]
It is ok for me. :+1:

**Comment by barthanssens** [comment]
Link to ontology was added, thanks for the review.

**Pull request merged**